docs/management/managing-tags.asciidoc

```diff
@@ -0,0 +1,70 @@
+[role="xpack"]
+[[managing-tags]]
+== Tags
+
+Tags enable you to categorize your saved objects. You can then easily filter for related objects based on shared tags.
+
+To begin, open the main menu, click *Stack Management*, then click *Tags*.
+
+[role="screenshot"]
+image::images/tags/tag-management-section.png[Tags management section]
+
+[float]
+=== Required permissions
+
+Access to *Tags* requires the `Tag Management` {kib} privilege. To add the privilege, open the menu, 
+click *Stack Management*, then click *Roles*.
+
+In addition:
+
+* The `read` privilege allows you to assign tags to the saved objects for which you have write permission.
+* The `write` privilege enables you to create, edit, and delete tags.
+
+
+NOTE: having the `Tag Management` {kib} privilege is not required to
+view tags assigned on objects the user has `read` access to, or to filter objects by tags
+in {kib} applications or from the navigational search.
+
+[float]
+[[settings-create-tag]]
+=== Create a tag
+
+Create a tag to assign to your saved objects.
+
+. Click *Create tag*.
+
+. Enter a name and select a color for the new tag.
```

Comment:
We lost the thought about limitations on names.

. Enter a name and select a color for the new tag.
+ 
The name can include the characters ..... and cannot be longer than 50 characters.

If you use a "+" after the step, the numbering should be ok.